Week3/Homework/Js-exercises/totalCost.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+let calculateTotalPrice = function(cart) {
+  for (const value in cart) {
+    sum = sum + cart[value];
+  }
+
+  console.log(sum);
```

Comment:
Would like to introduce two new methods in JavaScript called `Object.keys()` and `Array.prototype.reduce()`. Have a look at them and see if you can implement this logic using both of them 🙂